### PR TITLE
chore: fix repository JSON contents not being linted

### PR DESCRIPTION
**Pull request by wolfy1339**

### CODE REVIEW

File: .github/workflows/prettier.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+on:
+  schedule:
+    # https://crontab.guru/#0_0_*_*_7
+    - cron: "0 0 * * 7"
```
Comment (wolfy1339) [inline .github/workflows/prettier.yml:4]: @gr2m This value doesn't seem to be liked by GitHub Actions.
I used Crontab Guru to create it
```
invalid `cron` attribute "0 0 * * 7"
```

**Review by gr2m — DISMISSED** [review]

**Review by gr2m — COMMENTED** [review]
why not add the `{payload-examples,payload-schemas}/**/.json` files to the existing `lint` and `lint:fix` scripts?

**Comment by wolfy1339** [comment]
> why not add the `{payload-examples,payload-schemas}/**/.json` files to the existing `lint` and `lint:fix` scripts?

They already are included.

The script makes sure they are consistently formatted.
> The difference between this script and running `prettier --write` on the files directly is that `prettier` preserves multiline objects.

**Comment by gr2m** [comment]
sorry I don't follow. If the files are already formatted when running `npm run lint:fix`, why the extra workflow?

And why the schedule trigger for the extra workflow? Wouldn't it make more sense to run it on push to the default branch?

**Comment by wolfy1339** [comment]
The script formats the JSON files consistently, by stripping them of all whitespace and newlines. So as to let prettier add all the necessary whitespace and newlines where needed
https://github.com/octokit/webhooks/blob/master/bin/docs/format-with-prettier.md

> And why the schedule trigger for the extra workflow? Wouldn't it make more sense to run it on push to the default branch?

I believed it would be too noisy if done on every push, and once a week seemed appropriate. I'm happy to make it run on push

**Comment by gr2m** [comment]
> The script formats the JSON files consistently, by stripping them of all whitespace and newlines. So as to let prettier add all the necessary whitespace and newlines where needed
> https://github.com/octokit/webhooks/blob/master/bin/docs/format-with-prettier.md

got it 👍🏼 

> I believed it would be too noisy if done on every push, and once a week seemed appropriate. I'm happy to make it run on push

maybe instead of a pull request we could just directly push to `main` and use a personal access token from @octokitbot, and make the account an admin on the account?

**Comment by wolfy1339** [comment]
Sure. That sounds alright

**Comment by gr2m** [comment]
I made @octokitbot an admin

**Review by gr2m — APPROVED** [review]

**Pull request merged**